File: neteasycloudcourse/src/main/java/debug/LauncherActivity.java
package debug;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import org.gmfbilu.neteasycloudcourse.MainActivity;
import org.gmfbilu.superapp.lib_base.base.BaseActivity;

public class LauncherActivity extends BaseActivity {


    @Override
    public void findViewById_setOnClickListener(Bundle savedInstanceState) {
        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();

    }

    @Override
    public int setLayout() {
        return 0;
    }

    @Override
    public void onClick(View v) {

    }
}

File: module_kotlin/src/main/java/debug/LauncherActivity.kt
package debug

import android.content.Intent
import android.os.Bundle
import android.view.View
import org.gmfbilu.superapp.lib_base.base.BaseActivity
import org.gmfbilu.superapp.module_kotlin.MainActivity

class LauncherActivity : BaseActivity() {

    override fun onClick(v: View?) {
    }

    override fun findViewById_setOnClickListener(savedInstanceState: Bundle?) {
        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }

    override fun setLayout(): Int {
        return 0
    }
}